### ISSUE TITLE
Give an option to select months and year only

### ISSUE THREAD

**Issue by bisam-rd** [vaadin/vaadin-date-picker#566]
### Description

It would be nice if the date-picker had an option to disable day selection and only displayed months
like in vaadin-8.
In our business lots of calculation are run between ends of months so it does not make sense to pick a day.

**Comment by tomivirkki** [comment]
Thanks for the feature suggestion. Changing the resolution of `<vaadin-date-picker>` has been discussed earlier within the team and would be a nice addition to the component, it's not in our short-term roadmap though. Requires some design work beforehand.

`<vaadin-dropdown-menu>` could be considered as a workaround meanwhile.

**Comment by juuso-vaadin** [comment]
Here's a suggestion for the month resolution design.

![Lumo-month-picker_090818](https://user-images.githubusercontent.com/19607782/67465197-5e81f280-f64d-11e9-9a61-d1be16d0f1a6.png)

**Comment by aetasoul** [comment]
Any updates on this feature?

**Comment by web-padawan** [comment]
Currently it's still not in our short-term plans, but we might consider this in future.

**Comment by aetasoul** [comment]
Thank you!

**Comment by rolfsmeds** [comment]
Frankly I'm having trouble seeing how this would be better than a two field combination?

The DatePicker's benefits are 1. no need to type a date, and 2. it allows you to pick a date based on weekday etc if you don't know the exact date itself.

Neither of those two are an issue if you just need month+year.

**Comment by aetasoul** [comment]
@rolfsmeds In my case the problem is that the user need to select year/month only and now he must select also a random day. 
I have created two different combobox, one for the year and one for the month, so the problem is "solved".

It would be great to have the year/month selection directly in the DatePicker because aesthetically it is better and also in terms of accessibility it is better. 

Consider to have one simple page with a chart where you need to select the year and the month. I prefer to choose it in one place, with the DatePicker, not in two different view/place/location.

This is my point of view, I hope it could help understand why i'd like to have it.

**Comment by rolfsmeds** [comment]
I agree it might look nicer with a single field, but in terms of accessibility, two separate fields are much easier to understand, much easier to navigate by keyboard, not to speak of screen readers etc. It does mean pressing tab in between to get to the next field, but even that can be automated by programmatically focusing the month field after year is selected. What you lose in that one tab press, you win back by not having to scroll as much or navigate an arbitrary 2-dimensional picker with arrow keys.

The calendar-based picker raises the learning threshold and sacrifices some accessibility in order to make it quick and easy to pick dates that are 1) close to the starting point, and 2) especially in cases where you don't know the exact date but just want to pick something like "the first monday last month". But 2) is not a thing without dates, and 1) is just as quick with separate dropdowns.

I'm not a fan of the proposed design in this ticket as it repeats the months of each year, which is pointless as they are the same each year. A single dropdown with two separately scrollable panes would make more sense, but I'm having trouble seeing how that would be better than two separate dropdowns.

So in summary I actually consider your current solution superior to the proposed one, but I'm certainly still open to the idea if I can get an example of how a single dropdown would be better in terms of accessibility.

**Comment by aetasoul** [comment]
I also consider the end user part and in my case as accessibility they better understand the DatePicker.
They really don't know what is keyboard navigation, so it is useless in my case manage the focus and all the "automation".
I consider it better in term of accessibility because I prefer to find what I search in one place and not in different place.

Considering the coding part, I appreciate to declare just one object that can manage directly the Date. I don't like to declare two object, get the value and compose the final date.
